lib/core/stores/local.ts

```diff
@@ -354,7 +330,7 @@ export class LocalStore {
   }
 
   private async createPromiseFromSchedule(schedule: Schedule): Promise<DurablePromise | undefined> {
-    return this.localPromiseStore.create(
+    return this.promises.create(
       schedule.id,
```

Comment:
`schedule.promiseId,`

The promise id is different than the schedule id, we also need to expand it as it can contain template variables, for example:
```
promise/{{id}}/{{timestamp}}
```

See [here](https://github.com/resonatehq/resonate/blob/main/internal/app/coroutines/schedulePromises.go#L85-L87). This uses something called go templates so we'll need to do something similar client side